app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DownvotesController < ApplicationController
+  def create
+    if current_user
+      @review = Review.find(params[:review_id])
+      @vote = @review.votes.find_or_create_by(user: current_user)
```

Comment:
you could ask Seal Team 6 for some more hints on how to make this happen. I bet Dodi and Kylee would be willing to walk you through it.